lib/jsonapi/query_parser.ex

```diff
@@ -138,7 +150,7 @@ defmodule JSONAPI.QueryParser do
   def build_sort("", field), do: [asc: field]
   def build_sort("-", field), do: [desc: field]
 
-  def parse_include(config, ""), do: config
+  def parse_include(config, []), do: config
   def parse_include(%Config{} = config, include_str) do
     includes = handle_include(include_str, config)
     Map.put(config, :includes, includes)
```

Comment:
@jeregrine Seems like this should be `include` rather than includes but I'm hesitant to change it so it doesn't break other people's code. Thoughts?

Reply:
@StareIntoTheBeard lets support both then? Take include and includes, merge them and return :include and :includes with the same value?

This way we can be backwards and forward compatible?

Reply:
We could raise a deprecation warning for `includes`?

Reply:
@doomspork yeah that's a great idea.